.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This _should_ suggest that a user opening the project in VSCode to install Prettier and ESLint extensions.